### ISSUE TITLE
Change all ICD-10 CM operators to ICD-10 Conditions operators in Amgen's Algorithm Repository

### ISSUE THREAD

**Issue by aguynamedryan** [outcomesinsights/conceptql#144]
Amgen wants us to replace all instances of the ICD 10 CM operator to ICD-10 Conditions operator for all algorithms in their repository.  They want us to do it since manually updating this code could be error-prone.  They are aware a new patch will be required.

Assigning to @aguynamedryan to estimate work/hours.  Assigning to @markdanese for prioritization.  All time worked on this will be counted towards Amgen's Jigsaw support hours.

Reference outcomesinsights/amgen_jigsaw_external#124 for Amgen's request.

---
Moved from outcomesinsights/jigsaw#411
/cc @jenniferduryea @markdanese @aguynamedryan

**Comment by jenniferduryea** [comment]
To test the conversion, I created two algorithms (one with icd10cm and icd10 conditions) prior to patch.  Then evaluated the algorithms after patch was updated.  Confirmed that the icd10cm operator was now querying both vocabs 34 and 70.  I also confirmed that the icd10 conditions algorithm did not have sql code and had the following JSON code:
![image](https://user-images.githubusercontent.com/7853238/29796957-fa125016-8c08-11e7-8021-5046f03cef65.png)

I believe this is sufficient.  OI (Jen) is going to work with AMG (Fangfei) to ensure that all algorithms using ICD10 Condition operators will be moved over to ICD10CM prior to the patch update.  But, if there are any algorithms that fall through the cracks, it is nice to know the JSON code, with the available ICD10 Condition operator information, is available for QC purposes.

I also confirmed from the two algorithms above worked as expected in the Study Builder.  The ICD10CM algorithm showed a check mark, eligible for inclusion, while the ICD10 Conditions algorithm could not be added to the study builder.  See screenshot below:
![image](https://user-images.githubusercontent.com/7853238/29796993-3edc794c-8c09-11e7-8e87-1df80ece0116.png)

I also was able to create, run, and successfully complete a study using the ICD10CM algorithm against synpuf.  I also confirmed I was able to create and save a ICD10CM algorithm after the patch was applied.  I confirmed it was querying both 34 and 70 vocabularies.  Looks good!
